### DISCUSSION TITLE
Emojis to report customer sentiment

### DISCUSSION THREAD

**Discussion by srgupta-twilio** [twilio-labs/paste#3712] [Request: Icon] [answered]
### Description

HI, we are trying to report customer sentiment to agents during and after a conversation. We would like to use emojis for quick readability.
The happy face emoji is already available in Paste, requesting to add one for neutral and one for negative sentiment.

![image](https://github.com/twilio-labs/paste/assets/122378627/bf245803-60dd-4253-b78d-daa5404c02e5)

<img width="532" alt="Screenshot 2024-01-10 at 14 12 51" src="https://github.com/twilio-labs/paste/assets/122378627/d2d14968-ef86-4c96-bc13-fdcc5f0a9f97">

### Link to icon

https://www.streamlinehq.com/icons/core-line/mail/smiley/sad-face

### Icon type

- [x] UI icon
- [ ] Product icon

**Comment by SiTaggart** [comment]
Hi @srgupta-twilio 

Thanks for the dependency request. Sarah is out for a couple of weeks, but the ticket is in the backlog ready to refined upon her return.

I'm suggesting we add three new icons to represent positive, negative and neutral sentiment. We can reuse the emoji icon for positive, but they should be stand alone for their own semantic purposes.

Thanks

**Reply by srgupta-twilio** [reply]
That's perfect! Thanks Simon :)

**Reply by serifluous** [reply]
Hey @srgupta-twilio ! I'll send you the SVGs if your team wants to follow the self-service process here and get the icons added ASAP https://paste.twilio.design/introduction/contributing/icons#self-service-instructions .

If your engineering team doesn't want to go the self-service route, let us know, and we'll add the task to our backlog, which will take a bit longer.

**Reply by TheSisb** [reply]
Hi @srgupta-twilio,

Just following up here if this was still needed by your team. Let us know if you do, otherwise I will close this ticket out in ~48hrs

**Comment by srgupta-twilio** [comment]
Hi, our engineering team has decided to build it themselves and contribute
the icon to Paste. Thank you so much for sharing final icons with us.

**Reply by serifluous** [reply, marked as answer]
Hey @srgupta-twilio! We ended up adding the icons through this PR: #3808

**Comment by srgupta-twilio** [comment]
Thank you :)
